README.md

```diff
@@ -86,101 +143,121 @@ The only required section is `site_info`
   }
 }
 ```
-This section is used to setup your WordPress site for the first time. You can change most of these settings with the WordPress admin once you have logged in after setup.
 
-You can define a specific version of WordPress that you would like to use with `wordpress_version`. If you do not specify, it will use the latest release of WordPress.
+Make sure you change these values, especially `url` and `title` to something specific to your app. At this time, these are the only options managed by the build process.
+
+#### Updating WordPress
 
-Plugins and themes are configured in the same way. You can provide either:
- * `name` and specific `version`
- * `name` only and latest version will be used
- * `url` only
+By default, this package will install the latest version of WordPress. To update, run `cf restage APP_NAME`. cloud.gov will install a fresh copy of WordPress Core each time you `push` or `restage` so we **do not recommend** using the admin interface to manage updates to your site.
+
+If you want to pin your Core installation to a specific version of WordPress that you can. Add `"wordpress_version": VERSION_NUMBER` on line above the `site_info` section.
```

Comment:
s/that you can/you can/
s/on line/on a line/

Reply:
✅